stackle_app/package.json

```diff
@@ -1,9 +1,33 @@
 {
   "name": "stackleapp",
+  "version": "1.0.0",
+  "description": "Web Communication portal for OpenSource organizations to discuss project ideas and issues ",
   "private": true,
   "scripts": {
     "start": "gulp serve"
   },
+ "repository": {
+    "type": "git",
+    "url": "git+https://github.com/scorelab/Stackle.git"
+  },
+  "keywords": [
+    "stackle",
+    "SCORE"
+  ],
+  "author": "SCoRe Lab",
+  "license": "ISC",
```

Comment:
@scorelab/stackle-maintainers What should be the licence?

Reply:
Apache License Version 2.0